contact_map/contact_map.py

```diff
@@ -686,9 +637,15 @@ class ContactFrequency(ContactObject):
     """
     # Default for use_atom_slice, None tries to be smart
     _class_use_atom_slice = None
+    _pending_dep_msg = (
+        "ContactFrequency will be renamed to ContactMap in version 0.8. "
+        + "Invoking it as ContactFrequency will be deprecated in 0.8. For "
+        + "more, see https://github.com/dwhswenson/contact_map/issues/82"
```

Comment:
plusses are not needed inside brackets
```suggestion
        "Invoking it as ContactFrequency will be deprecated in 0.8. For "
        "more, see https://github.com/dwhswenson/contact_map/issues/82"
```
Lead to better indentation (in my opinion)